docs/tutorial/index.rst

```diff
@@ -282,6 +285,159 @@ you will access the build logs,
 otherwise it will take you directly to the documentation.
 When you are satisfied, you can merge the pull request!
 
+Customizing the build process
+-----------------------------
+
+The Settings page of the :term:`project home` allows you
+to change some *global* configuration values of your project.
+In addition, you can further customize the building process
+using the ``.readthedocs.yaml`` :doc:`configuration file </config-file/v2>`.
+This has two advantages:
+
+- The configuration lives next to your code and documentation, tracked by version control.
+- It can be different for every version (more on versioning in the next section).
+
+Read the Docs works without this configuration by making some decisions on your behalf.
+For example, what Python version to use, how to install the requirements, and others.
+
+Upgrading the Python version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For example, to explicitly use Python 3.9 to build your project,
+navigate to your GitHub repository, click on the :guilabel:`Add file` button,
+and add a ``.readthedocs.yaml`` file with these contents to the root of your project:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+   version: 2
+
+   build:
+     image: testing
+
+   python:
+     version: "3.9"
+
+The purpose of each key is:
+
+``version``
+  Mandatory, specifies :doc:`version 2 of the configuration file </config-file/v2>`.
+
+``build.image``
+  Specifies a newer :ref:`Docker image <builds:Docker images>`,
+  needed to use more recent versions of Python.
```

Comment:
maybe we can use 3.8 to avoid mentioning the `build.image`, I think of this more like a hack, hopefully it won't be needed soon.

Reply:
Good point, I considered it as a way to showcase `build.image`, but in light of upcoming changes #8453, better to simplify it.